Samples/Perl/query.pl

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use REST::Client;
+use MIME::Base64;
+use URI::Encode qw(uri_encode);
+use JSON::Parse ':all';
+ 
+################################################################################
+# Configuration
+################################################################################
+ 
+my $username = 'domain\username';  # user/pass works too for local SW accounts
+my $password = 'password_here';
+
+my $protocol = 'https';  # http or https will work here
```

Comment:
Port 17778 is always HTTPS, never HTTP. Is there some Perl magic that makes it auto-switch?